docs/aggregation-layer/v0.3.5/CompatibilityTable.md

```diff
@@ -0,0 +1,30 @@
+# Compatibility Table
+
+The following compatibility table shows which components should be used depending on the chain you are working with. This table helps ensure that the correct configurations and contracts are selected for each setup:
+
+|    Chain     |  verifierType   |        SC consensus         |  Proofs/Prover   |       Client       | AgglayerBridgeL2 | AgglayerGERL2 + AggOracle | AggOracleCommittee  | AggSender | Provers |
+|:------------:|:---------------:|:---------------------------:|:----------------:|:------------------:|:----------------:|:-------------------------:|:-------------------:|:---------:|:-------:|
+|    zkEVM     | StateTransition |      [PolygonZkEVMEtrog](https://github.com/agglayer/agglayer-contracts/blob/v12.1.5/contracts/consensus/zkEVM/PolygonZkEVMEtrog.sol)    |   FEP # Hermez   |  legacy-cdk-erigon |        ❌        |            ❌             |           ❌        |     ❌    |    ❌   |
+|  Valididum   | StateTransition |    [PolygonValidiumEtrog](https://github.com/agglayer/agglayer-contracts/blob/v12.1.5/contracts/consensus/validium/PolygonValidiumEtrog.sol)     |   FEP # Hermez   |  legacy-cdk-erigon |        ❌        |            ❌             |           ❌        |     ❌    |    ❌   |
+| v0.3.0-ECDSA |    ALGateway    |   [AggchainECDSAMultisig](https://github.com/agglayer/agglayer-contracts/blob/v12.1.5/contracts/aggchains/AggchainECDSAMultisig.sol)     |     PP # SP1     |  legacy-cdk-erigon |        ❌        |            ❌             |           ❌        |     ✅    |    ❌   |
```

Comment:
`v0.3.0-ECDSA` naming is old. We can set `v0.3.5-ECDSA`.
Same for `v0.3.0-FEP` --> `v0.3.5-FEP`